modules/database/sqlite/db.copy

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Plugin Name: Performance Lab - SQLite module.
+ * Plugin URI: https://github.com/WordPress/performance
+ * Description: Performance plugin from the WordPress Performance Team, which is a collection of standalone performance modules.
+ * Requires at least: 6.0
+ * Requires PHP: 5.6
+ * Version: 1.7.0
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ * Text Domain: performance-lab
+ *
+ * This file is auto-generated and copied from the performance-lab plugin.
+ * Please don't edit this file directly.
+ *
+ * @package performance-lab
+ */
```

Comment:
Can we reduce this header so that it only has the headers that are relevant for a drop-in? See https://github.com/johnbillion/query-monitor/blob/develop/wp-content/db.php#L1 for example. Having all the extra headers here is a bit of unnecessary maintenance.

Reply:
Of course! Done 👍